test/controllers/DirectDebitControllerSpec.scala

```diff
@@ -158,4 +158,14 @@ class DirectDebitControllerSpec extends ControllerBaseSpec {
       }
     }
   }
+
+  "the user is insolvent and not continuing to trade" should {
+
+    "return 403 (Forbidden)" in new DirectDebitDetailsTest {
+      override val authResult: Future[~[Enrolments, Option[AffinityGroup]]] = successfulAuthResult
```

Comment:
No need to override `authResult` as it's already set to `successfulAuthResult` by default in the trait